frontend/src/pages/organization/AccessManagementPage/components/OrgIdentityTab/components/IdentitySection/IdentityLdapAuthForm.tsx

```diff
@@ -143,6 +147,12 @@ export const IdentityLdapAuthForm = ({
   const { mutateAsync: updateMutateAsync } = useUpdateIdentityLdapAuth();
   const [tabValue, setTabValue] = useState<IdentityFormTab>(IdentityFormTab.Configuration);
   const { data: templates } = useGetAvailableTemplates(MachineIdentityAuthMethod.LDAP);
```

Comment:
behind the scenes, this checks for `list templates` permission. Perhaps we can do an `OR` in the API?